src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusService.scala

```diff
@@ -114,6 +114,6 @@ class ConsignmentStatusService(
 }
 
 object ConsignmentStatusService {
-  val validStatusTypes: Set[String] = Set("Series", "TransferAgreement", "Upload", "ConfirmTransfer", "Export", "ClientChecks")
+  val validStatusTypes: Set[String] = Set("Series", "TransferAgreement", "Upload", "ConfirmTransfer", "Export", "ClientChecks", "ServerFFID", "ServerChecksum", "ServerAntivirus")
```

Comment:
Is it worth popping in a unit test for both `validStatusTypes` and `validStatusValues` to check the elements they contain?